darts/models/forecasting/tft_model.py

```diff
@@ -871,6 +871,33 @@ def __init__(
         ----------
         .. [1] https://arxiv.org/pdf/1912.09363.pdf
         ..[2] Shazeer, Noam, "GLU Variants Improve Transformer", 2020. arVix https://arxiv.org/abs/2002.05202.
+
+        Examples
+        --------
+        >>> from darts.datasets import WeatherDataset
+        >>> from darts.models import TFTModel
+        >>> series = WeatherDataset().load()
+        >>> # predicting atmospheric pressure
+        >>> target = series['p (mbar)'][:100]
+        >>> # past observed rainfall (pretending to be unknown beyond index 100)
+        >>> past_cov = series['rain (mm)'][:100]
+        >>> # future temperatures (pretending this component is a forecast)
+        >>> future_cov = series['T (degC)'][:106]
+        >>> model = TFTModel(
+        >>>             input_chunk_length=6,
+        >>>             output_chunk_length=6,
+        >>>             n_epochs=5,
+        >>>             )
+        >>> # future_covariates is mandatory for `TFTModel`
+        >>> model.fit(target, past_covariates=past_cov, future_covariates=future_cov)
```

Comment:
TFTModel is using QuantileRegression by default. We should explain this and take num_samples>>1 and `pred.all_values()`

Reply:
Let me know if the explanations are detailed enough

Reply:
👍